app/javascript/dashboard/helper/auditlogHelper.js

```diff
@@ -89,11 +89,10 @@ function handleAccountUserCreate(auditLogItem, translationPayload, agentList) {
 }
 
 function handleAccountUserUpdate(auditLogItem, translationPayload, agentList) {
-  if (auditLogItem.user_id !== auditLogItem.auditable.user_id) {
-    translationPayload.user = getAgentName(
-      auditLogItem.auditable.user_id,
-      agentList
-    );
+  if (auditLogItem.user_id !== auditLogItem.auditable?.user_id) {
+    translationPayload.user = auditLogItem.auditable
+      ? getAgentName(auditLogItem.auditable?.user_id, agentList)
+      : 'deleted user';
```

Comment:
Can we use i18n here?
```suggestion
      :  this.$t(`AUDIT_LOGS.DELETED_USER`);
      
```

Reply:
@vishnu-narayanan We can't do i18n in a helper file.

Reply:
Done https://github.com/chatwoot/chatwoot/pull/9884/commits/711b0ce87a04127174a74e9306780bfe477fde59